### ISSUE TITLE
文字检测只可以检测出一类

### ISSUE THREAD

**Issue by butterniu** [PaddlePaddle/PaddleOCR#7360]
我训练了文件检测模型，总共大约包含2中背景图，但是只有在一个背景图中可以检测出数字的位置，用模型检测另一类的训练集中的数据发现仍无法识别出位置，训练数据我都重新检查了一遍完全 没有问题，大约这两中数据比例是1：1，共计5000张左右，训练了100轮，请问是否出现过类似的问题

file:///home/swls/work_dir/github/paddle/PaddleOCR.bak2/output/det_r18_vd_db/det_results/144_filp.jpg
这是可以识别的类型图片

file:///home/swls/work_dir/github/paddle/PaddleOCR.bak2/train_data/boing_number/img/boyin_0610/28_filp.jpg
这是不可以识别出的图片

**Comment by butterniu** [comment]
![144_filp](https://user-images.githubusercontent.com/44352254/186637667-bc36b27c-1ab8-4608-810e-339825f19b56.jpg)
这是可以识别的类型图片

**Comment by butterniu** [comment]
![144_filp](https://user-images.githubusercontent.com/44352254/186638087-bb9f7b11-8879-4ba2-b316-afb2f3d49e70.jpg)
这是可以识别的类型图片

**Comment by butterniu** [comment]
![52_filp](https://user-images.githubusercontent.com/44352254/186638157-187d9ba6-d3fa-4fae-b2f5-b738cc3df16e.jpg)
这是不可以识别出的图片

**Comment by butterniu** [comment]
这是我的配置文件

`
Global:
  algorithm: DB
  use_gpu: true
  epoch_num: 100
  log_smooth_window: 20
  print_batch_step: 20
  save_model_dir: /mnt/disk2/model/
  # distort: true
  save_epoch_step: 10
  eval_batch_step: 200000000
  train_batch_size_per_card: 4
  test_batch_size_per_card: 1
  image_shape: [3, 640, 640]
  reader_yml: ./configs/det/det_db_icdar15_reader_boeingnum.yml
  pretrain_weights: ./pretrain_models/ResNet18_vd_pretrained/
  # pretrain_weights: /mnt/disk2/model/model/iter_epoch_198
  save_res_path: ./output/det_r18_vd_db/predicts_db.txt
  checkpoints: /mnt/disk2/model/iter_epoch_100
  # checkpoints: /mnt/disk2/model1/model/iter_epoch_198
  # infer_img: ./train_data/boing_number/testImg/
  infer_img: /home/swls/work_dir/github/paddle/PaddleOCR.bak2/train_data/boing_number/img/boyin_0610/1634185753496.jpg
  save_inference_dir: /mnt/disk2/model/model


Architecture:
  function: ppocr.modeling.architectures.det_model,DetModel

Backbone:
  function: ppocr.modeling.backbones.det_resnet_vd,ResNet
  layers: 18

Head:
  function: ppocr.modeling.heads.det_db_head,DBHead
  model_name: large
  k: 50
  inner_channels: 256
  out_channels: 2

Loss:
  function: ppocr.modeling.losses.det_db_loss,DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  function: ppocr.optimizer,AdamDecay
  base_lr: 0.001
  beta1: 0.9
  beta2: 0.999
  decay:                                                                                                                         
    function: cosine_decay_warmup
    step_each_epoch: 32                                                                                                         
    total_epoch: 200

PostProcess:
  function: ppocr.postprocess.db_postprocess,DBPostProcess
  thresh: 0.3
  box_thresh: 0.1
  max_candidates: 1000
  unclip_ratio: 1.5
 








TrainReader:
  reader_function: ppocr.data.det.dataset_traversal,TrainReader
  process_function: ppocr.data.det.db_process,DBProcessTrain
  num_workers: 8
  img_set_dir: ./train_data/boing_number/img/
  label_file_path: ./train_data/boing_number/img/train.txt

EvalReader:
  reader_function: ppocr.data.det.dataset_traversal,EvalTestReader
  process_function: ppocr.data.det.db_process,DBProcessTest
  img_set_dir: ./train_data/boing_number/img/
  label_file_path: ./train_data/boing_number/img/test.txt
  test_image_shape: [768, 768]
  
TestReader:
  reader_function: ppocr.data.det.dataset_traversal,EvalTestReader
  process_function: ppocr.data.det.db_process,DBProcessTest
  infer_img:
  img_set_dir: ./train_data/boing_number/img/
  label_file_path: ./train_data/boing_number/img/test.txt
  test_image_shape: [768, 768]
  do_eval: True






`

**Comment by littletomatodonkey** [comment]
这块之前没遇到过，只用其中一类数据训练看下呢？
